ibis/expr/types/numeric.py

```diff
@@ -526,38 +526,6 @@ def bucket(
             include_over=include_over,
         ).to_expr()
 
-    def histogram(
```

Comment:
This is a breaking change since it deletes a method, not a refactor.

- Are we fine with it being a breaking change? (I think it's fine, it's a rare method and this is much cleaner)
- Can you update the message so it shows up as a breaking change in the release notes?

Reply:
Is this not showing up somewhere? I see it in the commit body.

Reply:
Oh, I thought the breaking change thing required something in the commit title too, not just a breaking change in the body. NVM, this is fine.

Reply:
Shouldn't we have `table.histogram()` implementation here instead? It would remain backward compatible.